src/components/client/campaign-application/helpers/campaignApplication.types.ts

```diff
@@ -0,0 +1,13 @@
+export type Step = {
+  title: string
+  component: JSX.Element
+}
+
+export enum Steps {
+  NONE = -1,
+  ORGANIZER = 0,
+  CAMPAIGN = 1,
+  CAMPAIGN_DETAILS = 2,
+}
+
+export type CampaignApplicationFormData = {}
```

Comment:
Linter errors on this. 
Change the CampaignApplicationFormData type temporary to `Record<string,string>` until the FormData body is figured out.

Reply:
No worries. This is what the CI/CD  is for.